frontend/src/app/layout.tsx

```diff
@@ -20,6 +21,7 @@ export default function RootLayout({ children }: PropsWithChildren) {
     <html lang="en">
       <body className={kongtext.className}>
         <WalletProvider>{children}</WalletProvider>
+        <Analytics />
```

Comment:
what does this for?

Reply:
will remove for now (this is to track Visitors once we deploy this repo)